packages/connect-migrate/src/migrate.ts

```diff
@@ -19,50 +19,179 @@ import { run } from "./run";
 import type { Transform } from "jscodeshift";
 import jscodeshift from "jscodeshift/src/core";
 import { Logger } from "./log";
+import { minVersion, lt } from "semver";
 
+// TODO: The following versions are hardcoded for now but we probably want to
+// be able to pull this information from somewhere eventually
 export const replacementMap = {
-  "@bufbuild/connect": "@connectrpc/connect",
-  "@bufbuild/connect-web": "@connectrpc/connect-web",
-  "@bufbuild/connect-fastify": "@connectrpc/connect-fastify",
-  "@bufbuild/connect-node": "@connectrpc/connect-node",
-  "@bufbuild/connect-next": "@connectrpc/connect-next",
-  "@bufbuild/connect-express": "@connectrpc/connect-express",
-  "@bufbuild/protoc-gen-connect-es": "@connectrpc/protoc-gen-connect-es",
-  "@bufbuild/protoc-gen-connect-web": "@connectrpc/protoc-gen-connect-web",
-  "@bufbuild/connect-query": "@connectrpc/connect-query",
-  "@bufbuild/protoc-gen-connect-query": "@connectrpc/protoc-gen-connect-query",
-  "@bufbuild/protoc-gen-connect-query-react":
-    "@connectrpc/protoc-gen-connect-query-react",
+  "@bufbuild/connect": {
+    newPackage: "@connectrpc/connect",
+    version: "0.13.1",
```

Comment:
We're setting the exact constraint`0.13.1`, which means that `npm update` won't update a minor version, and the next connect-migrate command is guaranteed to need --force-update.

By default, package managers use `^`. Can we do that too?